autogpt_platform/backend/migrations/20250222014114_add_email_verified_flag_to_user_table/migration.sql

```diff
@@ -0,0 +1,10 @@
+-- First, add the column as nullable to avoid issues with existing rows
+ALTER TABLE "User" ADD COLUMN "emailVerified" BOOLEAN;
+
+-- Set default values for existing rows
+UPDATE "User" SET "emailVerified" = true;
+
+-- Now make it NOT NULL and set the default
+ALTER TABLE "User" ALTER COLUMN "emailVerified" SET NOT NULL;
+ALTER TABLE "User" ALTER COLUMN "emailVerified" SET DEFAULT true;
```

Comment:
this was after a discussion with john that we want to trust all existing users, and trust new users going forward until we get stuff working

Reply:
So why do we need to verify at the first place ?

Reply:
because if we bounce an email we need to stop emailing that person, we're basically doing a tradeoff of checking valid now vs on first email. Because our customers all signed up on a form with their specific email to get access its likely most of the email addresses are real. Later on we will add a step to the signup and this will default false

Reply:
I know this sounds crazy lol